app/scripts/controllers/quota.js

```diff
@@ -32,6 +32,7 @@ angular.module('openshiftConsole')
     var limitRangesVersion = APIService.getPreferredVersion('limitranges');
 
     var watches = [];
+    var DEFAULT_POLL_INTERVAL = 60 * 1000; // milliseconds
```

Comment:
We should prob make this a `.constant()` since we have this `DEFAULT_POLL_INTERVAL` in a few places around the app now.